packages/stripe-graphql/src/schema/payment-method.ts

```diff
@@ -0,0 +1,10 @@
+import { builder } from '../builder'
+
+builder.objectType('StripePaymentMethod', {
+  fields: (t) => ({
+    id: t.exposeString('id'),
+    object: t.exposeString('object'),
+    created: t.exposeInt('created'),
+    livemode: t.exposeBoolean('livemode')
```

Comment:
likely would want to expose type as well (`type: t.expose({ type: PaymentMethodTypes })`)

Reply:
Hmm, I can't seem to get enums to work here. Do I need to add the `StripePaymentMethodTypes` to the SchemaBuilder somehow?

![CleanShot 2022-09-07 at 09 19 12](https://user-images.githubusercontent.com/331818/188814236-419861e2-d5e2-41f0-8c32-47a4454813e1.png)

Reply:
I managed to get it to work like this:

![CleanShot 2022-09-07 at 09 53 42](https://user-images.githubusercontent.com/331818/188821984-e20844d2-5c29-4b4f-8125-be2920fa1d32.png)

Reply:
You probably shouldn't add the enum to the builder like that.  That's shouldn't work either... Weird.   The issue here is that the type property is not compatible with that enum for some reason.  

Instead what I would do is use t.field with a resolver.  In the resolver you can either cast the type as the correct value (or any), or do some conversation or check to ensure the value is what you want.  Using t.field with a resolver will also give you a better error message, when there is a type compatibility issue t.expose has pretty useless errors, that specific type error is one of the worst parts of Pothos.   It might be something as simple as type being nullable